### PR TITLE
[ON HOLD] Added Locust Testing

### PR DESCRIPTION
**Pull request by bernardolk**
Now we have the option of doing a swarm testing using the Locust package.

`pip install locust` with your environment active.
Then, from the root, do `locust -f tests/locustfile.py`.
This will start a local server serving in port 8089, where you can find a dashboard by using your browser.
There you can set the # of concurrent users, alongside other settings.

In the locust file, you can also tweak the size of each payload and the number of different batches to be generated.
Each batch is generated using random spectra from a big MGF file containing 5760 spectra (so the hard limit is MAX_DIFFERENT_BATCHES * BATCH_SIZE = 5760 for this file, for now, as we could just load the file back again and keep generating new batches, but anyway)
We generate different batches/payloads to avoid caching in the Seldon pod, and assuring we will stress test the pod's memory usage.

 [ON HOLD] - we should move the tests to the omigami core repo. The public client shouldn't have tests that hit our dev/prod clusters. Once we do that, then we can "move that PR" to the other repo

### CODE REVIEW

**Comment by carloszanella** [comment]
Please add the description on how to run it in the README in a new section

File: tests/locustfile.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+MAX_DIFFERENT_BATCHES = 5
+BATCH_SIZE = 1000
+TOKEN = ENV["token"].get()
+MGF_PATH = str(ASSETS_DIR / "GNPS-NIST14-MATCHES.mgf")
+PREDICT_ENDPOINT = (
+    "https://mlops.datarevenue.com/seldon/seldon/spec2vec/api/v0.1/predictions"
+)
+
+# spec2vec client
+spec2vec = Spec2VecClient("no-need-for-token-here")
```
Comment (PierreMkt) [inline tests/locustfile.py:25]: why don't we need the admin token for the dev cluster here ? Did we disable auth for the prediction endpoint on the dev cluster ?

**Review by PierreMkt — APPROVED** [review]
Very cool, thanks Bernardo! Just one small question

File: tests/locustfile.py (hunk 1 above)
Comment (bernardolk) [reply tests/locustfile.py:25]: Its used in the post request itself, because here we don't use the client's post but the locust post

Comment (bernardolk) [reply tests/locustfile.py:25]: So, we actually use the token just not here : ]

**Pull request closed without merging**